umh.docs.umh.app/content/en/docs/features/datainfrastructure/unified-namespace.md

```diff
@@ -1,67 +1,114 @@
+or message brokers, we believe that a message broker approach is most suitable
+for most manufacturing applications. Consequently, every piece of information
+within the United Manufacturing Hub is transmitted via a message broker.
 
 ## What can I do with it?
 
-The Unified Namespace / Message Broker in the United Manufacturing Hub provides several notable functionalities in addition to the features already mentioned:
-
-- Easy integration using MQTT: Many modern shopfloor equipment can send and receive data using the MQTT protocol.
-- Easy integration with legacy equipment: Using tools like [Node-RED](/docs/architecture/microservices/core/node-red/), data can be easily extracted from various protocols such as Siemens S7, OPC-UA, or Modbus
-- Get notified in real-time via MQTT: The Unified Namespace allows you to receive real-time notifications via MQTT when new messages are published. This can be useful for applications that require near real-time processing of data, such as an AGV waiting for new commands.
-- Retrieve past messages from Kafka logs: By looking into the Kafka logs, you can always be aware of the last messages that have been sent to a topic. This allows you to replay certain scenarios for troubleshooting or testing purposes.
-- Efficiently process messages from millions of devices: The Unified Namespace is designed to handle messages from millions of devices in your factory, even over unreliable connections. By using Kafka, you can efficiently at-least-once process each message, ensuring that each message arrives at-least-once (1 or more times).
-- Trace messages through the system: The Unified Namespace provides tracing capabilities, allowing you to understand where messages are coming from and where they go. This can be useful for debugging and troubleshooting purposes. You can use the [Management Console](https://mgmt.docs.umh.app/docs/) to visualize the flow of messages through the system.to visualize the flow of messages through the system.
+The Unified Namespace in the United Manufacturing Hub allows for diverse functionalities and applications:
+
+- **Seamless Integration with MQTT**: Facilitates straightforward connection
+  with modern industrial equipment using the MQTT protocol.
+- **Legacy Equipment Compatibility**: Provides easy integration with older
+  systems using tools like [Node-RED](/docs/architecture/data-infrastructure/unified-namespace/node-red/)
+  or [Benthos UMH](/docs/features/connectivity/benthos-umh/),
+  supporting various protocols like Siemens S7, OPC-UA, and Modbus.
+- **Real-time Notifications**: Enables instant alerting and data transmission
+  through MQTT, crucial for time-sensitive operations.
+- **Historical Data Access**: Offers the ability to view and analyze past
+  messages stored in Kafka logs, which is essential for troubleshooting and
+  understanding historical trends.
+- **Scalable Message Processing**: Designed to handle vast quantities of data
```

Comment:
"vast quantities" --> chatgpt

maybe write something like "by using IT standard tools we can theiretically process GB/s seconds. in IT it is not measured anymore by messages/second, but instead in GB/s". something like that.